packages/react-server-test-pages/pages/bottleneck/dataRequests.js

```diff
@@ -0,0 +1,12 @@
+import {ReactServerAgent} from "react-server";
+
+export default class DataRequestsPage {
+
+	getElements() {
+		//Perform 1 million data requests before returning
+		for (var i = 0; i < 999999; i++) {
+			ReactServerAgent.get('/data/delay');
+		}
+		return (<div>Data requests complete!</div>);
+	}
+}
```

Comment:
Yeah, these should get kicked off in `handleRoute`, and then we should make root elements wait for them.

Ideally we could style the root elements so they all fit above the fold and affect the WPT speed index.